### DISCUSSION TITLE
Is there a function in Rust regex to calculate the size of the compiled regex？

### DISCUSSION THREAD

**Discussion by yangwentong** [rust-lang/regex#939] [Q&A] [answered]
I want to calculate the size of the compiled regex.
`ProgramSize` is provided in RE2 to calculate the size.

**Comment by BurntSushi** [comment, marked as answer]
Nope.

**Reply by yangwentong** [reply]
I would like to know why this function is not implemented?

**Reply by BurntSushi** [reply]
Because nobody has done it. Nobody has asked for it. And nobody has given a good reason for why it should exist.

**Reply by yangwentong** [reply]
thank you
